tests/unit/vic_run/test_snow_utility.py

```diff
@@ -2,11 +2,11 @@
 from vic.vic import ffi
 from vic import lib as vic_lib
 
-
+# updated to include veg-dependent albedo
 def test_snow_albedo_new_snow():
     assert vic_lib.snow_albedo(
         0.1, 1., 0.7, -77, 3600., 0,
-        ffi.cast('char', False)) == vic_lib.param.SNOW_NEW_SNOW_ALB
+        ffi.cast('char', False)) <= vic_lib.param.SNOW_NEW_SNOW_ALB
```

Comment:
I think you want to write a test that checks that the result of `snow_albedo` on timestep 0 is `max_snow_albedo`. You may also consider adding an additional test for when MAX_SNOW_ALBEDO is true or false.

Reply:
@bartnijssen and I just discussed this and I'm going to update this test in a future PR just so we can go ahead and merge the PR. I just created an issue for it; see #843 for reference.

Reply:
Okay. It sounds like you two have a plan in place. I'll just say that I think the best time/place to write tests is in the PR that changes specific bits of code. Its why we have a checkbox in the PR template for `new tests added`.